src/main/java/com/hierynomus/ntlm/functions/NtlmFunctions.java

```diff
@@ -36,6 +40,8 @@
 
     static final byte[] LMOWFv1_SECRET = new byte[]{0x4B, 0x47, 0x53, 0x21, 0x40, 0x23, 0x24, 0x25}; // KGS!@#$%
 
+    public static final SecureRandom RANDOM = new SecureRandom();
```

Comment:
It might make sense to be able to inject a different 'random provider'. This will ease testing so that we can test against a "predictable random value".